commands/Moderation/unmute.js

```diff
@@ -0,0 +1,28 @@
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			permissionLevel: 6,
+			requiredPermissions: ['MANAGE_ROLES'],
+			runIn: ['text'],
+
+			description: 'Unmutes a mentioned user.',
+			usage: '<member:user> [reason:string] [...]',
+			usageDelim: ' '
+		});
+	}
+
+	async run(msg, [user, ...reason]) {
+		const member = msg.guild.members.get(user.id) || await msg.guild.members.fetch(user).catch(() => null);
```

Comment:
Remove the get, fetch tries to get from cache, and fetches from the API if it doesn't exist.